Install-Guide/Community_Packages.md

```diff
@@ -5,59 +5,59 @@
+|OpenSUSE Build|SLES12SP3     |     X     |     X     |     X     |
+|Service[4]    |Leap42.3      |     X     |     X     |     X     |
+|              |Leap15        |     X     |     —     |     —     |
+|              |SLES15        |     X     |     —     |     —     |
+|              |Tumbleweed    |     X     |     —     |     —     |
 
 
 #### Related Packages
 
-|              |              | gdeploy | gluster-block | glusterfs-coreutils | nfs-ganesha | Samba |
-|--------------|--------------|:-------:|:-------------:|:-------------------:|:-----------:|:-----:|
-|CentOS Storage|el6           |    X    |       X       |         X           |      X      |   ?   |
-|SIG           |el7           |    X    |       X       |         X           |      X      |   ?   |
-|              |              |         |               |                     |             |       |
-|Fedora        |F26           |    X    |       X       |         X           |      X      |   ?   |
-|              |F27           |    X    |       X       |         X           |      X      |   ?   |
-|              |F28           |    X    |       X       |         X           |      X      |   ?   |
-|              |F29(rawhide)  |    X    |       X       |         X           |      X      |   ?   |
-|              |              |         |               |                     |             |       |
-|Debian        |Jessie/8      |    —    |       —       |         X           |      X      |   ?   |
-|              |Stretch/9     |    —    |       —       |         X           |      X      |   ?   |
-|              |Buster/10(Sid)|    —    |       —       |         X           |      X      |   ?   |
-|Ubuntu        |Trusty/14.04  |    —    |       —       |         X           |      X      |   ?   |
-|Launchpad     |Xenial/16.04  |    —    |       —       |         X           |      X      |   ?   |
-|              |Artful/17.10  |    —    |       —       |         X           |      X      |   ?   |
-|              |Bionic/18.04  |    —    |       —       |         X           |      X      |   ?   |
-|              |Cosmic/18.10  |    —    |       —       |         X           |      X      |   ?   |
-|              |              |         |               |                     |             |       |
-|OpenSuSE Build|SLES12SP3     |    X    |     X         |         X           |      X      |   ?   |
-|Service       |Leap42.3      |    X    |     X         |         X           |      X      |   ?   |
-|              |Leap15        |    X    |     X         |         X           |      X      |   ?   |
-|              |SLES15        |    X    |     X         |         X           |      X      |   ?   |
-|              |Tumbleweed    |    X    |     X         |         X           |      X      |   ?   |
+|              |              | gdeploy | gluster- | glusterfs- | nfs-ganesha | storhaug | Samba |
```

Comment:
This table isn't rendered properly, at least on [github](https://github.com/kalebskeithley/glusterdocs/blob/ee78720bb285a86931ef19aee9d04efaac92525b/Install-Guide/Community_Packages.md)